blocks/v2-footer/v2-footer.css

```diff
@@ -0,0 +1,590 @@
+    width: 90%;
+    flex-wrap: wrap;
+    gap: 10px;
+    margin-top: 40px;
+  }
+  
+  .v2-footer .v2-footer-bottom-section li a:hover {
+    color: var(--c-primary-white);
+  }
+  
+  .v2-footer .footer-form-and-social-section {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .v2-footer .v2-footer-bottom-section {
+    margin-bottom: 38px;
+    border-top: 1px solid var(--c-tertiary-cool-gray);
+    width: 100%;
+    margin-top:5vh;
+  }
+
+  .v2-footer .v2-footer-links-section {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    grid-area: v2-content;
+  }
+
+  .v2-footer .v2-footer-links-section ul {
+    min-width: 50%;
+  }
+  
+  .v2-footer .v2-footer-form-header .v2-footer__title {
+    font-family: var(--ff-body);
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 16px;
+    letter-spacing: 1.92px;
+    text-transform: uppercase;
+    margin-bottom: 12px;
+  }
+
+  .v2-footer .v2-footer-main-content {
+    display: grid; 
+    grid-template-columns: 1fr; 
+    grid-template-rows: 1fr 1fr 1fr 1fr; 
+    gap: 0 0; 
+    /* stylelint-disable-next-line declaration-block-no-redundant-longhand-properties */
+    grid-template-areas: 
+      "v2-logo"
+      "v2-content"
+      "v2-form"
+      "v2-social"; 
+      height: 65vh;
+  }
+  
+  .v2-footer-social-section { 
+    grid-area: v2-social;
+    margin-top: 0;
+  }
+
+  .v2-footer-form-section { 
+    grid-area: v2-form;
+    height: 15vh;
+  }
+
+  .v2-footer-logo-section { 
+    grid-area: v2-logo; 
+  }
+
+  .v2-footer .v2-footer-links-section ul li {
+    margin-bottom: 16px;
+  }
+
+  /* Prefooter */
+  .v2-footer .v2-pre-footer-content {
+    background-color: var(--c-primary-gold);
+  }
+
+  .v2-footer .v2-pre-footer-content ul {
+    display: flex;
+    width: 100%;
+    padding: 32px 16px;
+    justify-content: space-around;
+    align-items: center;
+    margin: 0;
+    flex-direction: column;
+  }
+
+  .v2-footer .v2-pre-footer-content ul li {
+    font-size: 24px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 140%; /* 33.6px */
+    letter-spacing: 0.48px;
+    color:var(--c-primary-black);
+    position: relative;
+    width: 100%;
+    border-bottom: solid black 1px;
+    padding: 24px 0;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .v2-footer .v2-pre-footer-content ul li:last-child {
+    border-bottom: none;
+  }
+
+  .v2-footer .v2-pre-footer .v2-pre-footer-content ul li a {
+    color: var(--c-primary-black);
+  }
+
+  .v2-footer .v2-pre-footer .v2-pre-footer-content ul li span  svg{ 
+    height: 24px;
+    width: 24px;
+  }
+
+  .v2-footer .v2-pre-footer-list-wrapper {
+    display: flex;
+    width: 100%;
+    padding: 24px;
+    justify-content: flex-start;
+    align-items: baseline;
+    background-color: var(--c-secondary-graphite);
+  }
+
+  .v2-footer .v2-pre-footer-list-wrapper h3 {
+    color: var(--primary-white-ffffff, #FFF);
```

Comment:
where is this colour coming from? `--primary-white-ffffff` I think that we should use `--c-primary-white`, check in styles please

Reply:
Fixed